nerfstudio/configs/method_configs.py

```diff
@@ -637,6 +641,14 @@
                 lr_final=5e-7, max_steps=30000, warmup_steps=1000, lr_pre_warmup=0
             ),
         },
+        "field_background": {
+            "optimizer": AdamOptimizerConfig(lr=4e-3, eps=1e-15),
+            "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-4, max_steps=30000),
+        },
+        "appearance_embed": {
```

Comment:
Are these being added to normal splatfacto? Any particular reason why?

Reply:
Yes. Because enables the bg model in splatfacto would also possibly gives a better outdoor scene.
![train](https://github.com/nerfstudio-project/nerfstudio/assets/50019703/7d493350-dade-41a1-a1db-ee7a25660cd1)
![egypt](https://github.com/nerfstudio-project/nerfstudio/assets/50019703/abfd9c25-6d34-4b18-bb7e-da10be5d7235)